pkg/reading/fields.go

```diff
@@ -0,0 +1,5 @@
+package reading
+
+type Fields struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type Fields should have comment or be unexported](https://app.codacy.com/app/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4570750)